Makefile.backport

```diff
@@ -423,7 +454,7 @@ RHEL_OSV_NAME = $(shell cat $(KLIB_BUILD)/include/generated/uapi/linux/version.h
 SLES_OSV_NAME = $(shell cat $(KLIB_BUILD)/include/generated/autoconf.h | grep "CONFIG_SUSE_KERNEL " | cut -d " " -f 2 | cut -d "_" -f 2)
 ifeq ($(RHEL_OSV_NAME), RHEL)
 	OS_DISTRO_NAME := $(RHEL_OSV_NAME)
-else ifneq ($(CUSTOM_KERN_1_VER), "")
+else ifneq ($(CUSTOM_KERN_1_VER),)
 	OS_DISTRO_NAME := $(CUSTOM_KERN_1_VER)
```

Comment:
Please check binary rpm for SLES , Issue has been re introduced.

Reply:
Done